ERCS/eip-draft-erc-bts.md

```diff
@@ -0,0 +1,139 @@
+---
+title: ERC-BTS (Basket Token Standard)
```

Comment:
```suggestion
title: Basket Token
```
You don't need to include ERC in the title.  
It is suggested not to include the word standard, this is implied by being an ERC.